.github/workflows/Dockerfile

```
@@ -1,16 +1,21 @@
 FROM centos as centos-python
 RUN yum install -y make gcc openssl-devel libffi-devel sqlite-devel glibc-langpack-en
+RUN useradd -ms /bin/bash buildout
+USER buildout
 ARG PYTHON_VER
 ENV LC_ALL C.UTF-8
-RUN mkdir buildout
-WORKDIR /buildout
-COPY Makefile Makefile.configure /buildout/
+RUN mkdir /home/buildout/sandbox
+WORKDIR /home/buildout/sandbox
+COPY Makefile Makefile.configure /home/buildout/sandbox/
 RUN make PYTHON_VER=${PYTHON_VER} download_python
 RUN make PYTHON_VER=${PYTHON_VER} python
 FROM centos-python
 ARG PYTHON_VER
-COPY doc /buildout/doc
-COPY src /buildout/src
-COPY zc.recipe.egg_ /buildout/zc.recipe.egg_
-COPY setup.* dev.py *.rst *.txt buildout.cfg .coveragerc /buildout/
+COPY doc /home/buildout/sandbox/doc
+COPY src /home/buildout/sandbox/src
+COPY zc.recipe.egg_ /home/buildout/sandbox/zc.recipe.egg_
+COPY setup.* dev.py *.rst *.txt buildout.cfg .coveragerc /home/buildout/sandbox/
+USER root
+RUN chown -R buildout:buildout *
```

What's the current working directory at this point?  I don't remember if `FROM ...` resets the entire state, or if a previous `WORKDIR` persists.

Anyway for readability's sake I suggest 
```suggestion
RUN chown -R buildout:buildout /home/buildout/sandbox
```

---

ok